pkg/component/gardeneraccess/access.go

```diff
@@ -122,6 +123,26 @@ func (g *gardener) Destroy(ctx context.Context) error {
 	return managedresources.DeleteForShoot(ctx, g.client, g.namespace, ManagedResourceName)
 }
 
+var (
+	// TimeoutWaitForManagedResource is the timeout used while waiting for the ManagedResources to become healthy
+	// or deleted.
+	TimeoutWaitForManagedResource = 1 * time.Minute
```

Comment:
Almost every where in our code base we have used time out of 2 minutes and in some special cases 5 minutes. Any specific reason to keep timeout 1 minute here?
```suggestion
	TimeoutWaitForManagedResource = 2 * time.Minute
```